components/BioBox.js

```diff
@@ -39,11 +39,10 @@ const Bio = styled.div`
     flex: 1;
     min-width: 20%;
     img {
+      display: block;
+      margin: 0 auto 1.6rem auto;
       width: 100%;
-
-      @media (min-width: ${props => props.theme.medium}) {
-        float: left;
-      }
+      max-width: ${props => props.theme.small};
```

Comment:
I updated the styles on headshots for folks with bios on the about us page. Previously, at tablet-sized viewports, not having a `max-width` meant that these headshots were huge. This makes them a little more reasonable.

### Before

<img width="765" alt="Screen Shot 2020-06-27 at 5 03 06 PM" src="https://user-images.githubusercontent.com/7942714/85934556-d5385500-b898-11ea-8a12-3ddfee9017e1.png">

### After

<img width="764" alt="Screen Shot 2020-06-27 at 5 02 54 PM" src="https://user-images.githubusercontent.com/7942714/85934557-d8cbdc00-b898-11ea-8c73-a3f9210afdfe.png">